### PR TITLE
Update to 2.023.3

### PR DESCRIPTION
**Pull request by isuruf**
cc @saraedum

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+{% set sha256 = "1ab0297b6ea1516066300bc30f8db6c3c6e66f5d2cfce7cb8ffe38b65d089724" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.bz2
-  url: http://mirrors.mit.edu/sage/spkg/upstream/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  url:  https://salsa.debian.org/science-team/sympow/-/archive/upstream/{{ version }}/sympow-upstream-{{ version }}.tar.gz
```
Comment (isuruf) [inline recipe/meta.yaml:10]: Is this under the same license? https://salsa.debian.org/science-team/sympow/blob/upstream/README.md#forked-code has only a copyright.

Comment (saraedum) [reply recipe/meta.yaml:10]: yes, it's the same adhoc license as before. just the copyright has changed.

**Comment by saraedum** [comment]
@isuruf: This looks good to me now. There are some warnings during the testing
```
**WARNING** failed to create data bin package cache folder /home/conda/feedstock_root/build_artifacts/sympow_1533253022288/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/share/cache/sympow/datafiles/le64
```
I have not really checked what is going on here but sympow appears to work fine (for Sage's purposes at least.)

**Review by saraedum — APPROVED** [review]

**Comment by isuruf** [comment]
conda doesn't keep around empty directories and mkdir fails because the parent directories are not there.

**Comment by isuruf** [comment]
What do you think about doing something like `touch $PREFIX/share/cache/sympow/datafiles/`

**Comment by isuruf** [comment]
```
mkdir -p $PREFIX/share/cache/sympow/datafiles/
touch $PREFIX/share/cache/sympow/datafiles/empty
```

**Comment by saraedum** [comment]
Sure, would you mind touching `.keep` instead?

**Pull request merged**